tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py

```diff
@@ -95,25 +119,236 @@ def _is_whitespace(c):
+                Those features are model-specific and can be crafted from :class:`~transformers.data.processors.squad.SquadExample`
+                using the :method:`~transformers.data.processors.squad.squad_convert_examples_to_features` method.
+
+                Args:
+                    input_ids: Indices of input sequence tokens in the vocabulary.
+                    attention_mask: Mask to avoid performing attention on padding token indices.
+                    token_type_ids: Segment token indices to indicate first and second portions of the inputs.
+                    cls_index: the index of the CLS token.
+                    p_mask: Mask identifying tokens that can be answers vs. tokens that cannot.
+                        Mask with 1 for tokens than cannot be in the answer and 0 for token that can be in an answer
+                    example_index: the index of the example
+                    unique_id: The unique Feature identifier
+                    paragraph_len: The length of the context
+                    token_is_max_context: List of booleans identifying which tokens have their maximum context in this feature object.
+                        If a token does not have their maximum context in this feature object, it means that another feature object
+                        has more information related to that token and should be prioritized over this feature for that token.
+                    tokens: list of tokens corresponding to the input ids
+                    token_to_orig_map: mapping between the tokens and the original text, needed in order to identify the answer.
+                    start_position: start of the answer token index
+                    end_position: end of the answer token index
+                """
+                idx = example_index
+                identifier = ['input_ids_{}'.format(idx), 'input_mask_{}'.format(idx), 'segment_ids_{}'.format(idx)]
+
+                annotation = ExtendedQuestionAnsweringAnnotation(
+                    identifier,
+                    np.array(unique_id),
+                    np.array(span["input_ids"]),
+                    np.array(span["attention_mask"]),
+                    np.array(span["token_type_ids"]),
+                    np.array(cls_index),
+                    p_mask,
+                    answers[example_index],
+                    example["context_text"],
+                    example["tokens"],
+                    example['is_impossible'],
+                    span["paragraph_len"],
+                    span["token_is_max_context"],
+                    span["tokens"],
+                    span["token_to_orig_map"],
+                    start_position,
+                    end_position
+                )
+                annotations.append(annotation)
+                unique_id += 1
+
+            # if (example_index + 1 ) % 101 == 0:
+            #     break
+            if example_index % 100 == 0:
+                print(example_index)
```

Comment:
There's even a debug print there - is this a WIP PR?

Reply:
yes, it is WIP and it is my turn to finish it :)